src/xtopcom/xbasic/xbasic_size.hpp

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2017-2018 Telos Foundation & contributors
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#pragma once
+
+#include <stdlib.h>
+#include <stdio.h>
+#include <string>
+#include <iostream>
+
+#include "xbase/xns_macro.h"
```

Comment:
from .hpp perspective, only <string> & xns_macro.h is required. other headers are useless